### PR TITLE
Fix issues in Quick Starts.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/quick-starts/oidc-backchannel-logout-sample.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ If you have your own application, click the button below.
 
 ----
 
+## Set up the sample application
+
+Follow the below provided steps to set up the Playground application.
```
Comment (gomathyk) [inline en/docs/quick-starts/oidc-backchannel-logout-sample.md:14]:
```suggestion
Follow the steps given below to set up the Playground application.
```

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 2]
```diff
@@ -15,53 +15,28 @@ The user is required to log in to two different applications, `application1` and
...
+1. Access the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`).
+
+2. Navigate to **Main** > **Identity Providers** > **Resident Identity Provider**.
+
+3. Expand the **Inbound Authentication Configuration** section.
+
+4. Select **SAML2 Web SSO Configuration** and **Download SAML Metadata**.
 
-## Configure the sample
+    ![Download SAML metadata](../../assets/img/samples/download-saml-metadata.png)
```
Comment (gomathyk) [inline en/docs/quick-starts/saml-front-channel-logout.md:30]: Is this aligned properly?

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 3]
```diff
@@ -15,53 +15,28 @@ The user is required to log in to two different applications, `application1` and
...
-2.  Rename the downloaded metadata file obtained from the above step as `wso2_abc.xml`.
-3.  Copy it to `<SAMPLE_HOME>/sample/src/main/resources/metadata`.
-4.  Open the `<SAMPLE_HOME>/sample/src/main/webapp/WEB-INF/securityContext.xml` file. 
-5.  Search for **bean id** with the tag as metadata (`bean id=”metadata”`). 
-6.  Under the `list` tag of the `metadata bean id` add the following xml configuration and save the file. Note that this has the link to the metadata file,`wso2_abc.xml`, that we generated and renamed. 
+2.  Copy the downloaded SAML metadata file to `<SAMPLE_HOME>/sample/src/main/resources/metadata`.
+3.  Open the `<SAMPLE_HOME>/sample/src/main/webapp/WEB-INF/securityContext.xml` file. 
+4.  Search for **bean id** with the tag as metadata (`bean id=”metadata”`). 
+5.  Under the `list` tag of the `metadata bean id` add the following xml configuration and save the file. Note that this has the link to the metadata file,`wso2.xml`, that we downloaded.
```
Comment (gomathyk) [inline en/docs/quick-starts/saml-front-channel-logout.md:39]:
```suggestion
5.  Under the `list` tag of the `metadata bean id`, add the following xml configuration and save the file. Note that this has the link to the metadata file,`wso2.xml`, that we downloaded. 
```

**Review by gomathyk — APPROVED** [review]

File: en/docs/quick-starts/saml-front-channel-logout.md (hunk 2 above)
Comment (dewniMW) [reply en/docs/quick-starts/saml-front-channel-logout.md:30]: yes

**Pull request merged**